lib/matplotlib/backend_tools.py

```diff
@@ -1020,6 +1021,42 @@ def _mouse_move(self, event):
         self.toolmanager.canvas.draw_idle()
 
 
+class ToolHelpBase(ToolBase):
+    description = 'Print tool list, shortcuts and description'
+    default_keymap = rcParams['keymap.help']
+    image = 'help.png'
+
+    def _format_tool_keymap(self, name):
+        keymaps = self.toolmanager.get_tool_keymap(name)
+        # Capitalize "ctrl+a" -> "Ctrl+A" but leave "a" as is.
+        return ", ".join(re.sub(r"\w{2,}|(?<=\+)\w",
+                                lambda m: m.group(0).capitalize(),
```

Comment:
as mentioned, `ctrl+a` -> `Ctrl+A` but also `ctrl+A` -> `Ctrl+Shift+A`.

Reply:
Please push the fix directly.

Reply:
@anntzer I cannot push to your branch (permission denied). Either we have to resolve the permissions, or you can cherry pick from https://github.com/timhoffm/matplotlib/commits/anntzer/helptool

Reply:
Dunno why, I ticked the "allow edits from maintainers".  In any case, I pulled it (thanks for the test) and further simplified format_shortcut (str.title is handy here...).